improvelib/initializer/config.py

```diff
@@ -369,14 +369,21 @@ def initialize_parameters(self,
         self.logger.debug("Current section: %s", section)
         self.logger.debug("Current section config parameters: %s", section_config)
         self.logger.debug("Updating config")
+
         # Overrides dictionary of defaults with config params
+        # breakpoint();
+        # from pprint import pprint
+        # pprint(self.cli.default_params)
         for cfp in section_config:
-            if cfp in self.final_params:
```

Comment:
@wilke This was a question left behind by Alex. Do you know if this is needed?

Reply:
@jonesse3 I assume, afaik Natasha added it.

Reply:
For now we can ignore this to merge the rest of the commits.